test-integration/examples/modifiers-today.test.tsx

```diff
@@ -13,10 +13,20 @@ beforeEach(() => {
   render(<Example />);
 });
 
+describe('when rendering a month that contains today', () => {
+  test('it should add the default class name for today', () => {
+    expect(getDayButton(today)).toHaveClass('rdp-day_today');
+  });
+  test('it should not add this class name for any day that is not today', () => {
+    const notToday = addDays(today, 1);
+    expect(getDayButton(notToday)).not.toHaveClass('rdp-day_today');
+  });
```

Comment:
This test is just checking that "tomorrow" doesn't have the class. Ideally here should be a `test.each` and run over all the days. As alternative, you could get the container from the setup render result, and check how many `.rdp-day_today` class are rendered 

Make the `setup` to return the container, [like here](https://github.com/gpbl/react-day-picker/blob/master/test-integration/examples/custom-day.test.tsx).

Then you can write instead:

```suggestion
test('the calendar should have only one ".rdp-day_today" class', () => {
    const todays = container.querySelectorAll('.rdp-day_today');
    expect(todays).toHaveLength(1);
});
```